docs/topics/spiders.rst

```diff
@@ -693,12 +694,17 @@ SitemapSpider
 
         We can define a ``sitemap_filter`` function to filter ``entries`` by date::
 
-            def sitemap_filter(entries):
-                from datetime import datetime
-                for entry in entries:
-                    date_time = datetime.strptime(entry['lastmod'], '%Y-%m-%d')
-                    if date_time.year >= 2005:
-                        yield entry
+            class FilteredSitemapSpider(scrapy.SitemapSpider):
```

Comment:
SitemapSpider is not exposed at top level, I think an import from scrapy.spiders is needed (see SitemapSpider examples section).

Reply:
Updated